etc/config.xml

```diff
@@ -35,14 +35,18 @@
                 <sepa_flow>sale</sepa_flow>
                 <split_payments_refund_strategy>1</split_payments_refund_strategy>
                 <return_path>checkout/cart</return_path>
-                <enable_oneclick>1</enable_oneclick>
+                <enable_oneclick>0</enable_oneclick>
                 <enable_recurring>0</enable_recurring>
+                <payment_pre_authorized>pending</payment_pre_authorized>
+                <fraud_manual_review_accept_status>pending</fraud_manual_review_accept_status>
+                <fraud_manual_review_status>pending</fraud_manual_review_status>
+                <send_email_bank_sepa_on_pending>No</send_email_bank_sepa_on_pending>
                 <group>adyen</group>
             </adyen_abstract>
             <adyen_cc>
-                <active>1</active>
+                <active>0</active>
                 <model>AdyenPaymentCcFacade</model>
-                <title>Credit Card</title>
+                <title>Adyen Credit Card</title>
```

Comment:
I believe this field will be visible by default to the shoppers. We don't want any adyen branding to shoppers so better leave this as it is.